EmployeeManagementSystem/PresentationLayer/Program.cs

```diff
@@ -0,0 +1,57 @@
+using Serilog;
+using PresentationLayer.Middlewares;
+using DataAccessLayer.Data;
+using Microsoft.EntityFrameworkCore;
+using DataAccessLayer.Interface;
+using DataAccessLayer.Repositories;
+using BusinessLogicLayer.Interface;
+using BusinessLogicLayer.Services;
+using PresentationLayer.Mapping;
+using BusinessLogicLayer.Mapping;
+
+var builder = WebApplication.CreateBuilder(args);
+
+//Logger
+Log.Logger = new LoggerConfiguration().MinimumLevel.Warning()
+    .WriteTo.File("log/logs.txt", rollingInterval: RollingInterval.Day).CreateLogger();
+builder.Host.UseSerilog();
+
+//DB connection
+builder.Services.AddDbContext<ApplicationDbContext>(options =>
+    options.UseSqlServer(builder.Configuration.GetConnectionString("DefaultConnection")));
+
+//Mapping Profiles
+builder.Services.AddAutoMapper(typeof(BusinessMappingProfile));
+builder.Services.AddAutoMapper(typeof(PresentationMappingProfile));
```

Comment:
why do you need to use AddAutoMapper() two times?

Reply:
I though that I needed that to keep layer architecture more strict(Data layer is not used in Presentation). But if I don't need that I will change it

Reply:
it depends on your needs, in general you can use two types of mapping if you want